content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Bare Metal provisioning on HPE GreenLake using Terraform
+date: 2023-02-26T19:38:34.667Z
+author: Chaita Mylarappachar
+authorimage: /img/chaitra-mylarappachar-192.png
+disable: false
+tags:
+  - hpe-greenlake
+  - terraform
+  - bare-metal-as-a-service
+  - hpegl
+  - BMaaS
+---
+# Introduction
+
+The HPE GreenLake for Private Cloud Enterprise: bare metal (BMaaS) service offering enables you to create dedicated compute instances deployed on a physical IT infrastructure facilitating on-demand scalability, convenience, and agility as a cloud service. 
+Using BMaaS, you can create compute-instances provisioned with specific operating systems, network connections, one or more public SSH keys, and optional network-attached storage volumes.
+The service can be accessed via GUI as well as via public APIs, enabling developers to use an Infrastructure-as-Code tool to build, change, and manage infrastructure in a consistent and repeatable way.
+
+## HPE GreenLake Terraform Provider
+
+The HPE GreenLake Terraform provider **hpegl** by HPE GreenLake provides Infrastructure-as-Code support for HPE GreenLake Cloud Services.
+Using the **hpegl** Terraform provider, you can automate the management of your infrastructure. You can provision OS on bare metal, spin virtual machines and bring up a Kubernetes cluster, starting right from bare metal all the way up in the stack to desired configurations and applications.
+
+In this blog post, I will walk you through the steps required to use the HPE GreenLake Terraform Provider to deploy and further manage bare metal compute instances.
+
+# Preparing for Infrastructure-as-Code implementation
+
+## Setting up API Client for access
```

Comment:
Recommend adding in "an" prior to "API" and changing "Client" to "client" (lower case initial letter)